kernel/arch/dreamcast/include/dc/vmu_fb.h

```diff
@@ -0,0 +1,122 @@
+/* KallistiOS ##version##
+
+   dc/vmu_fb.h
+   Copyright (C) 2023 Paul Cercueil
+
+*/
+
+#include <kos.h>
+
```

Comment:
There should also be a Doxygen comment here explaining briefly what this header file contains.